client/app.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      movieId: 1,
+      cast: []
+    };
+  }
+
+  // === GET CAST MEMBERS BASED ON MOVIE ID === //
```

Comment:
This all looks awesome! I need to practice writing my fetch functions in this less-verbose way--looks like I tack on a bunch of unnecessary code to mine.